ext/sockets/tests/socket_set_option_invalid_value_for_IPPROTO_TCP_TCP_CONGESTION.phpt

```diff
@@ -0,0 +1,33 @@
+--TEST--
+socket_set_option($socket, IPPROTO_TCP, TCP_CONGESTION, INVALID_TYPE_FOR_OPTION)
+--EXTENSIONS--
+sockets
+--SKIPIF--
+<?php
+
+if (!defined('IPPROTO_TCP')) {
```

Comment:
`IPPROTO_TCP` had been translated into `SOL_TCP` (to remove platform differences likely).

Reply:
See my run of your new tests

![image](https://github.com/user-attachments/assets/52f4aadf-dbb5-4002-9cf2-f09792abff12)